src/worker/views/most.ts

```diff
@@ -814,6 +814,85 @@ const updatePlayers = async (
 				};
 			};
 			after = tidAndSeasonToAbbrev;
+		} else if (type === "oldest_mvp" || type === "youngest_mvp") {
+			const oldest = type === "oldest_mvp";
+			title = `${oldest ? "Oldest" : "Youngest"} MVP`;
+			description = `These are the players who won their MVP at the ${
+				oldest ? "oldest" : "youngest"
+			} age ${oldest ? "mininum age of 30" : "maximum age of 28"}.`;
```

Comment:
No need to filter out by age, since all of these cap at 100 players anyway. Also no explicit age cap will better handle leagues with weird settings.